AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/capture/DefaultCameraCaptureSource.swift

```diff
@@ -222,7 +222,9 @@ import UIKit
         }
         // choose a supported format that is closest to `self.format`.
         try? captureDevice.lockForConfiguration()
-        let newAVFormat = captureDevice.formats.min { avFormatA, avFormatB in
+        let newAVFormat = captureDevice.formats.filter {
+            VideoCaptureFormat.fromAVCaptureDeviceFormat(format: $0).maxFrameRate == format.maxFrameRate
```

Comment:
lets do this after resolution? I think most people would prefer 720p at 15fps then 240p at 30 fps (some captures have a tendency to have higher framerates available for lower resolutions.)